packages/translation/src/server.ts

```diff
@@ -20,7 +20,7 @@ export async function requestTranslationsAPI<T>(
   const settings = ServerConnection.makeSettings();
   const requestUrl = URLExt.join(
     settings.baseUrl,
-    '/lab/api/translations/', // API Namespace
+    `${settings.appUrl}/api/translations/`, // API Namespace
```

Comment:
@blink1073 is the `api/translations`  part also something that is configurable and should be use accordingly?

Reply:
Seems reasonable, it would be handled [here](https://github.com/jupyterlab/jupyterlab/blob/9c5252392fa671949d67fe5aa2b83c78144e066b/packages/application/src/frontend.ts#L322)

Reply:
Oh, so what should I do :-) ?

Reply:
- Add a default path [here](https://github.com/jupyterlab/jupyterlab/blob/9c5252392fa671949d67fe5aa2b83c78144e066b/packages/application/src/lab.ts#L264).  
- Make sure `translationsApiUrl` is actually there in `PageConfig`.
- Use `app.urls.translationUrl` in your extension and pass it in as an option to this function.

Reply:
:+1: Thanks!

Reply:
Done @blink1073